share/spice/expand_url/expand_url.js

```diff
@@ -21,12 +22,16 @@
                 sourceUrl: "http://untiny.me/",
                 sourceName: "Untiny"
             },
+            normalize: function (item) {
+                return {
+                    title: item.org_url,
+                    url: item.org_url,
+                    description: "Expanded URL for: " + decodeURIComponent(query)
```

Comment:
@sahildua2305 Good catch; That's annoying. Can you add a CSS file to undo the text-transform here?

If we go with the description we'd need CSS to remove the extra space anyways...